plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeCdfPageSink.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.json.JsonCodec;
+import io.airlift.slice.Slice;
+import io.trino.filesystem.TrinoFileSystemFactory;
+import io.trino.hdfs.HdfsEnvironment;
+import io.trino.spi.PageIndexerFactory;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.type.Type;
+import io.trino.spi.type.TypeManager;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.OptionalInt;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.airlift.slice.Slices.wrappedBuffer;
+import static io.trino.plugin.deltalake.DeltaLakeColumnType.REGULAR;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+
+public class DeltaLakeCdfPageSink
+        extends AbstractDeltaLakePageSink
+{
+    public static final String CHANGE_TYPE_COLUMN_NAME = "_change_type";
+    public static final String CHANGE_DATA_FOLDER_NAME = "_change_data";
+
+    private final String tableLocation;
+
+    private final ImmutableList.Builder<DataFileInfo> dataFileInfos = ImmutableList.builder();
+
+    public DeltaLakeCdfPageSink(
+            List<DeltaLakeColumnHandle> inputColumns,
+            List<String> originalPartitionColumns,
+            PageIndexerFactory pageIndexerFactory,
+            HdfsEnvironment hdfsEnvironment,
+            TrinoFileSystemFactory fileSystemFactory,
+            int maxOpenWriters,
+            JsonCodec<DataFileInfo> dataFileInfoCodec,
+            String outputPath,
```

Comment:
It looks like `outputPath` has a different meaning in each of the Page Sinks. For `DeltaLakePageSink` it's the base table location but here it's the CDC file directory. Let's rename the variables so that the `tableLocation` is always the root... table location

If `outputPath` is a directory maybe we call it `outputDirectoryPath` as well. To clarify it's not a file path